vendor/_appmap/wrapt/wrappers.py

```diff
@@ -524,6 +525,7 @@ def __init__(self, wrapped, instance, wrapper, enabled=None,
         object.__setattr__(self, '_self_binding', binding)
         object.__setattr__(self, '_self_parent', parent)
         object.__setattr__(self, '_bfws', list())
+        object.__setattr__(self, "_appmap_instrumented", False)
```

Comment:
I'm not sure how I feel about modifying a vendored lib. I'm worried it'll be too easy to miss it when we update or if we ever decide to switch to non-vendored.

Reply:
This isn't the first change I've had to make to `wrapt`, so it's already the case that we have to merge from upstream, rather than just overwriting. Those other changes will also keep us from switching to non-vendored....